mage/query-modules/python/kmeans-clustering.md

```diff
@@ -0,0 +1,171 @@
+---
+id: kmeans
+title: kmeans
+sidebar_label: kmeans
+---
+
+import Tabs from '@theme/Tabs';
+import TabItem from '@theme/TabItem';
+
+export const Highlight = ({children, color}) => (
+<span
+style={{
+      backgroundColor: color,
+      borderRadius: '2px',
+      color: '#fff',
+      padding: '0.2rem',
+    }}>
+{children}
+</span>
+);
+
+[![docs-source](https://img.shields.io/badge/source-kmeans-FB6E00?logo=github&style=for-the-badge)](https://github.com/memgraph/mage/blob/main/python/kmeans.py)
+
+## Abstract
+
+The k-means algorithm clusters given data by trying to separate samples in `n` groups of equal variance, by minimizing a criterion known as 
+within-the-cluster sum-of-squares. To understand more about it, jump to the [algorithm](../../algorithms/machine-learning-graph-analytics/k-means-clustering-algorithm) page.
+
+| Trait               | Value                                                          |
+| ------------------- | -------------------------------------------------------------- |
+| **Module type**     | <Highlight color="#FB6E00">**module**</Highlight>              |
+| **Implementation**  | <Highlight color="#FB6E00">**Python**</Highlight>              |
+| **Graph direction** | <Highlight color="#FB6E00">**directed/undirected**</Highlight> |
+| **Edge weights**    | <Highlight color="#FB6E00">**weighted/unweighted**</Highlight> |
+| **Parallelism**     | <Highlight color="#FB6E00">**sequential**</Highlight>          |
+
+## Procedures
+
+### `get_clusters( n_clusters, embedding_property, init, n_init, max_iter, tol, algorithm, random_state)`
+This procedure returns for each node to which cluster it belongs to.
+
+#### Input:
+
+- `n_clusters : int` ➡ The number of clusters to form.
+- `embedding_property : str` ➡ Node property where embeddings are stored.
+- `init : str` ➡ Initialization method. If `k-means++` is selected, the initial cluster centroids are selected using sampling based on an empirical probability distribution of the points’ contribution to the overall inertia. This technique speeds up convergence and is theoretically proven to be `O(logk)`-optimal. 
+If `random` then `n_clusters` observations (rows) are chosen for the initial centroids randomly from the data.
+- `n_init : int` ➡ Number of times the k-means algorithm will be run with different centroid seeds.
+- `max_iter : int` ➡ Length of one walk in walk sampling.
+- `tol : float` ➡ Relative tolerance with regards to Frobenius norm of the difference in the cluster centers of two consecutive iterations to declare convergence.
+- `algorithm : str` ➡ Options are `lloyd`, `elkan`, `auto`, `full`.
+- `random_state : int` ➡ Use an int to make the randomness deterministic.
```

Comment:
```suggestion
- `random_state : int` ➡ Random seed for the algorithm.
```
By the way, does this parameter have a default value?

Reply:
I think it does! I will update it